pkg/scheduler/framework/runtime/framework.go

```diff
@@ -964,8 +964,7 @@ func (f *frameworkImpl) RunPermitPlugins(ctx context.Context, state *framework.C
 		status, timeout := f.runPermitPlugin(ctx, pl, state, pod, nodeName)
 		if !status.IsSuccess() {
 			if status.IsUnschedulable() {
-				msg := fmt.Sprintf("rejected pod %q by permit plugin %q: %v", pod.Name, pl.Name(), status.Message())
-				klog.V(4).Infof(msg)
+				klog.V(4).InfoS("Rejected pod by permit plugin", "pod", klog.KObj(pod), "plugin", pl.Name(), "status", status.Message())
```

Comment:
```suggestion
				klog.V(4).InfoS("Pod rejected by permit plugin", "pod", klog.KObj(pod), "plugin", pl.Name(), "status", status.Message())
```

is slightly better IMO :)

Reply:
I agree with you ,and i have fixed it,thank you very much.